app/lib/finder_api.rb

```diff
@@ -133,6 +133,7 @@ def query_builder_class
     "/services" => "services",
     "/transparency-and-freedom-of-information-releases" => "transparency_and_freedom_of_information_releases",
     "/all-content" => "all_content",
+    "/policy-and-engagement" => 'policy_and_engagement',
```

Comment:
We should adopt the latest user language, see https://www.gov.uk/search/advanced?group=policy_and_engagement&topic=%2Ftransport

```suggestion
    "/policy-papers-and-consultations" => 'policy-papers-and-consultations',
```